packages/client/src/raffles/types/postRaffleParticipation.types.ts

```diff
@@ -0,0 +1,23 @@
+import type { Config } from '../..';
+import type {
+  Raffle,
+  RaffleParticipation,
+  RaffleParticipationStatus,
+} from './raffles.types';
+
+export type PostParticipationData = {
+  userId: string;
+  billingAddressId: string;
+  shippingAddressId: string;
+  productVariantId: string;
+  paymentTokenId: string | null;
+  trackingCorrelationId: string | null;
```

Comment:
`trackingCorrelationId` is not required.

Reply:
done